src/main/java/racing/Car.java

```diff
@@ -0,0 +1,30 @@
+package racing;
+
+import java.util.Random;
+
+public class Car {
+    static String DISTANCE_STRING = "-";
```

Comment:
```suggestion
    private static final String DISTANCE_STRING = "-";
```

접근 제한자가 private이어도 괜찮지 않을까요?
그리고 상수를 의도하셨다면 변경할 수 없도록 final 한스푼 정도 첨가해주시면 좋아요.